app/access/models.py

```diff
@@ -16,9 +25,18 @@ def __str__(self) -> str:
     first_name = models.CharField(_(_context, "First name"))
     last_name = models.CharField(_(_context, "Last name"))
     email = models.EmailField(_(_context, "Email"))
+    deleted_at = models.DateTimeField(_(_context, "deleted at"), null=True, blank=True)
 
     provider = models.ForeignKey("provider.Provider", on_delete=models.CASCADE)
 
+    # By default only return active users
+    objects = ActiveUserManager()
+    all_objects = models.Manager()
```

Comment:
:thinking: Isn't it confusing to override a very common default behaviour like this?

If I do `User.objects` somewhere, I would expect to get access to all objects without any filter applied. If I wanted something specific, I would check if there is something dedicated for it, something like a `User.active_objects`. Then it's also clearly visible that here some filtering is happening. With `User.objects` I would be very confused not to get the full set.

I am not against the current solution, in a way it's elegant. Just saying that, if I had not been informed about this, I would probably be very confused not to get all objects when calling `User.objects`.

How do you see that? Maybe also @msom ?

Reply:
My thinking was that the "disable user" is really like a "delete user", so we usually don't want these users. The special case is that we need to get/recover the deleted users. I think if you are not informed about it, you might also be confused to receive deleted objects when calling `User.object`.

Reply:
Yeah right, also confusing to see users you assumed to be deleted. To me still a little less confusing because at least it behaves like every else: `objects` shows _all_ objects, not just a subset.

But good enough for me to keep it as it is :+1: